exonum/src/encoding/float.rs

```diff
@@ -65,10 +66,9 @@ impl F32 {
     /// assert!(val.is_none());
     /// ```
     pub fn try_from(value: f32) -> Option<Self> {
-        if value.is_normal() {
-            Some(Self { value })
-        } else {
-            None
+        match value.classify() {
+            FpCategory::Normal | FpCategory::Zero => Some(Self { value }),
+            _ => None,
```

Comment:
I propose to move this check into a function.

Reply:
I don't think it worth it.